readme.md

```diff
@@ -1,7 +1,7 @@
 # Product cards
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://Zaransoftline.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://Zaransoftline.github.io/layout_product-cards/report/html_report/)
```

Comment:
Also, need to add a test link to the PR description. And try to fix test

Reply:
didn't fix it

Reply:
fix previous comments